src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -23,10 +25,21 @@
 
 <br>
 <div class="form-group margin-0">
-  <div ${isNumResponsesMax ? 'class="col-sm-2 form-inline mobile-align-left"' : 'class="col-sm-4 form-inline mobile-align-left"'}
+  <div ${isNumResponsesMax ? 'class="col-sm-3 form-inline mobile-align-left"' : 'class="col-sm-5 form-inline mobile-align-left"'}
       ${isRecipientNameHidden ?  'style="display:none"' : 'style="text-align:right"'}>
 
-    <label for="input">To${isRecipientTeam ? ' Team' : ''}: </label>
+    <label for="input">
+      <span data-toggle="tooltip"
+        data-placement="top"
+        title="<%= Const.Tooltips.EVALUEE_DESCRIPTION %>">
+        Evaluee
+      </span>
+      <c:choose>
+        <c:when test="${isRecipientStudent}"> (Student)</c:when>
+        <c:when test="${isRecipientInstructor}"> (Instructor)</c:when>
+        <c:when test="${isRecipientTeam}"> (Team)</c:when>
```

Comment:
You can do a string compare here, as explained in [this article](https://stackoverflow.com/questions/5182727/jsp-accessing-enum-inside-jsp-el-tags).